lib/build.gradle

```diff
@@ -281,9 +281,21 @@ task renameLibkiwixSoFile(type: Copy) {
     }
 }
 
+task compileTestFile(type: Exec) {
+    workingDir "${projectDir}/src/test/"
+    commandLine 'javac', '-g', '-d', '.', '-s', '.', '-cp', 'junit-4.13.jar:' + buildDir.path +'/libs/*', 'test.java'
+}
+
+task runTests(type: Exec) {
+    workingDir "${projectDir}/src/test/"
+    dependsOn compileTestFile
+    commandLine 'java', '-Djava.library.path=' + buildDir.path, '-javaagent:jacoco-0.8.7/lib/jacocoagent.jar', '-cp', 'junit-4.13.jar:hamcrest-core-1.3.jar:' + buildDir.path +'/libs/*lib*.jar' + ':.', 'org.junit.runner.JUnitCore', 'test'
```

Comment:
Do we have to run `java` ourselves ?
gradle probably have specific target for testing. We should integrate our testing with it.